src/test/java/com/puppycrawl/tools/checkstyle/grammar/comments/AllBlockCommentsTest.java

```diff
@@ -96,9 +93,9 @@ public void init() {
         @Override
```

Comment:
Not for this line, but why do we need the line separators in the collection above? If we don't need them, it would be great to just do something like:
```java
IntStream.range(0, 62)
    .forEach(i -> COMMENTS.add(String.valueOf(i)));


```

Reply:
@nrmancuso done